### ISSUE TITLE
malfunction

### ISSUE THREAD

**Issue by KeyMap001** [AlexPfaltz/PfaltzUI#2]
When the player has several buffs

in the buff panel he does not select well to read the description.

**Comment by AlexPfaltz** [comment]
Yes, I've already noticed this bug, I don't understand why it appeared, everything should be correct in size. In principle, the buffs window is optional, as it duplicates information from the main screen. You can click on the buff icon next to the health bar, and a window with information about the buff will appear. That's why I completely removed the window with the list of buffs in an additional mod that expands the character window.

**Comment by KeyMap001** [comment]
> Yes, I've already noticed this bug, I don't understand why it appeared, everything should be correct in size. In principle, the buffs window is optional, as it duplicates information from the main screen. You can click on the buff icon next to the health bar, and a window with information about the buff will appear. That's why I completely removed the window with the list of buffs in an additional mod that expands the character window.

TRY: in controls 

<active_buff_entry>
<panel controller add width="75" height="77"
...
</active_buff_entry>

example:
<panel controller="ActiveBuffEntry" width="75" height="77" style="press, hover, scroll" sound="[recipe_click]" disableautobackground="true" sound_play_on_hover="[default_hover]">

It should work

**Comment by AlexPfaltz** [comment]
Yes, it works, thanks
